src/features/Header/components/Search/Search.jsx

```diff
@@ -0,0 +1,22 @@
+import styles from './Search.module.scss';
+
+const Search = () => (
+  <div className={styles.search_wrapper}>
+    <p className={styles.find_txt}>FIND YOUR MOVIE</p>
+    <form action="" className={styles.search_form}>
+      <input
+        type="text"
+        defaultValue="What do you want to watch?"
+        className={styles.input_txt}
+      />
+      <input
```

Comment:
buttons can be also moved to base as you generally will have 2 types of buttons - red and rey